python/labbox_ephys/helpers/prepare_snippets_h5.py

```diff
@@ -6,15 +6,15 @@
 import spikeextractors as se
 
 
-@hi.function('prepare_snippets_h5', '0.2.5')
+@hi.function('prepare_snippets_h5', '0.2.6')
 @hi.container('docker://magland/labbox-ephys-processing:0.3.19')
 @hi.local_modules(['../../labbox_ephys'])
 def prepare_snippets_h5(
     recording_object,
     sorting_object,
     start_frame=None,
     end_frame=None,
-    max_events_per_unit=1000,
+    max_events_per_unit=None,
```

Comment:
The default max_events_per_unit is now None, meaning that we include all events in the snippets file by default.